website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

```diff
@@ -48,7 +48,7 @@ The following fields are required when creating a Snowflake connection:
 
 | Field | Description | Examples |
 | ----- | ----------- | -------- |
-| Account | The Snowflake account to connect to.| `db5261993`, `db5261993.east-us-2.azure` |
+| Account | The Snowflake account to connect to. Take a look [here](https://docs.getdbt.com/reference/warehouse-profiles/snowflake-profile/#account) to determine what the account field should look like based on your region.| `db5261993`,`db5261993.east-us-2.azure` |
```

Comment:
If you use a relative link, then the docs site will validate that the links are all working in CI checks! 

```suggestion
| Account | The Snowflake account to connect to. Take a look [here](reference/warehouse-profiles/snowflake-profile/#account) to determine what the account field should look like based on your region.| `db5261993`,`db5261993.east-us-2.azure` |
```